.travis.yml

```diff
@@ -1,6 +1,6 @@
 # GENERATED, DO NOT EDIT!
 # To change, edit `build-support/bin/generate_travis_yml.py` and run:
-# ./pants --quiet run build-support/bin:generate_travis_yml > .travis.yml
+#   ./v2 run build-support/bin:generate_travis_yml > .travis.yml
```

Comment:
We couldn't use V2 run before because the stdout redirection would include Pants output.

Reply:
Related: are we ready to use v2 to bootstrap the root `pants.pex`?

Reply:
We are, but it's slow because of the PEX resolve. This isn't a big deal with local execution, but is bad for the CI bootstrap shards that run `./v2 binary pants.pex` every single CI run - it would slow down total wall time by ~1 minute.

Maybe we should do it for the sake of dogfooding? But I was waiting for us to fix the Pex resolve bug.